src/components/service-metrics/elasticsearch-service-metrics.njk

```diff
@@ -0,0 +1,193 @@
+{% extends "../services/_tabbed_layout.njk" %}
+{% from "govuk-frontend/govuk/components/tag/macro.njk" import govukTag %}
+{% from "./metric.macro.njk" import metric %}
+
+{% block pageTitle %}
+  {{ service.entity.name }} - Service Metrics
+{% endblock %}
+
+{% block insideTabs %}
+
+<div class="border-bottom-box">
+  <p>{{ govukTag({text: "experimental"}) }}</p>
+
+  <p class="govuk-body">
+    Backing service metrics is a new feature under active development.
+    Please email us at <a href="mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk" class="govuk-link">gov-uk-paas-support@digital.cabinet-office.gov.uk</a> if you have any feedback.
+  </p>
+</div>
+
+<div class="govuk-grid-column-full">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
```

Comment:
The class used here is different to the non-elasticsearch metrics, and it has some left-padding which makes things look visually weird. Compare to https://github.com/alphagov/paas-admin/pull/585/files#diff-351269a2a8e3efd8ef06dafa03fd7379R9.